src/Adapter/Backup/BackupRepository.php

```diff
@@ -42,24 +44,20 @@
      */
     public function get()
     {
-        $backupPath = opendir(PrestaShopBackup::getBackupPath());
-
-        if (false === $backupPath) {
-            return null;
-        }
+        $backupFinder = (new Finder())
+            ->files()
+            ->in(PrestaShopBackup::getBackupPath())
```

Comment:
maybe i could inject backup path via constructor and then move this service to `Core` namespace as `PrestaShopBackup::getBackupPath()` is the only legacy part?